packages/block-library/src/navigation-link/edit.js

```diff
@@ -304,30 +296,64 @@ function NavigationLinkEdit( {
 	);
 }
 
+/**
+ * Returns the color object matching the slug, or undefined.
+ *
+ * @param {Array}  colors      The editor settings colors array.
+ * @param {string} colorSlug   A string containing the color slug.
+ * @param {string} customColor A string containing the custom color value.
+ *
+ * @return {Object} Color object included in the editor settings colors, or Undefined.
+ */
+const getColorObjectByColorSlug = ( colors, colorSlug, customColor ) => {
+	if ( customColor ) {
+		return customColor;
+	}
+
+	if ( ! colors || ! colors.length ) {
+		return;
+	}
+
+	return get( find( colors, { slug: colorSlug } ), 'color' );
+};
+
 export default compose( [
 	withSelect( ( select, ownProps ) => {
 		const {
 			getBlockAttributes,
 			getClientIdsOfDescendants,
 			hasSelectedInnerBlock,
 			getBlockParentsByBlockName,
+			getSettings,
 		} = select( 'core/block-editor' );
 		const { clientId } = ownProps;
 		const rootBlock = head(
 			getBlockParentsByBlockName( clientId, 'core/navigation' )
 		);
-		const navigationBlockAttributes = getBlockAttributes( rootBlock );
+		const navBlockAttrs = getBlockAttributes( rootBlock );
```

Comment:
Is something preventing us from keeping it as `navigationBlockAttributes`?

Reply:
Since we've limited the columns per line which means that a large name makes the code kinda funny. Sometimes it requires to break the line in two or even more. Reducing it gets a little bit better improving the eligibility of the code, imho.

Reply:
I think that's fine, see https://github.com/WordPress/gutenberg/pull/19775#discussion_r374800524

Reply:
Ok, my answer at this moment is just doing a janitorial PR after this one., if we consider that worths it. Anyway, if we have a strong opinion that we should rename it, let's do it on it!

Reply:
Already renamed :-)